bottlecap/src/traces/trace_agent.rs

```diff
@@ -112,8 +112,8 @@ impl TraceAgent {
 
         let server = server_builder.serve(make_svc);
 
-        info!("Trace Agent started: listening on port {TRACE_AGENT_PORT}");
-        info!(
+        debug!("Trace Agent started: listening on port {TRACE_AGENT_PORT}");
```

Comment:
Should we collapse the 2 debugs in one line?

Reply:
eh, I like the newline between them when I check the logs. It's nice to see if the port got goofed or if the duration took longer than expected. It's just personal preference on my end though, if you think it should be changed I have no problem with that. Happy to slap a `\n` in here and call it good!